### ISSUE TITLE
set_axis_lock not working on KinematicBody 3D

### ISSUE THREAD

**Issue by MrCrowley2** [godotengine/godot#25798]
**Godot version:**
3.1 Beta 3

**OS/device including version:**
Kubuntu 18.04

**Issue description:**
The Axis lock is not working.
According to the docs I'd assume to call:

```{r tidy=FALSE}
set_axis_lock(move_lock_z,true) 
print("lock z " , get_axis_lock(move_lock_z))
```

The output on the console is : "lock z False"
And the body still moves on z axis by collision.

I tried some variations:
```{r tidy=FALSE}
self.set_axis_lock(PhysicsServer.BODY_AXIS_LINEAR_Z, true)
self.set_axis_lock(move_lock_z, true)
move_lock_z = true
PhysicsServer.body_set_axis_lock(self.get_rid(), PhysicsServer.BODY_AXIS_LINEAR_Z, true)
```

Neither of them results in a locked z axis.
For a 2.5D platformer z-axis lock would be crucial

the whole axis lock panel that was available in 3.06 in the inspector is gone as well in 3.1
Was the feature discarded?

**Minimal reproduction project:**
[AxisLockBug.zip](https://github.com/godotengine/godot/files/2851690/AxisLockBug.zip)

**Comment by QbieShay** [comment]
@AndreaCatania

**Comment by Favkis** [comment]
hey whole year passed, fix this bug maybe?

**Comment by Calinou** [comment]
@Favkis Godot is a community-developed project. Bugs are fixed when someone opens a pull request :slightly_smiling_face:

**Comment by TomWor** [comment]
That's an odd one. Anyone figure out a proper workaround for now at least?

**Comment by RichardEllicott** [comment]
> That's an odd one. Anyone figure out a proper workaround for now at least?
> 
> Edit: This seems to work for me:
> 
> ```gdscript
> self.transform.origin.z = 0
> ```

that is the same hack i have been using, it seems to be okay .... again yes also i am waiting for any sort of official recognition or promise of fixing this bug actually ever since i started with godot at version 3.0

i'm not sure of what the consequences of the hack may be but so far it does seem to have none

**Comment by madmiraal** [comment]
Fixed with #25798.
